rtic-monotonics/Cargo.toml

```diff
@@ -40,6 +41,9 @@ nrf5340-app-pac = { version = "0.12.2", optional = true }
 nrf5340-net-pac = { version = "0.12.2", optional = true }
 nrf9160-pac = { version = "0.12.2", optional = true }
 
+# STM32
+stm32-metapac = { version = "13.0.0", features = ["metadata"], optional = true }
```

Comment:
note `metadata` is supposed to be used from `build.rs` only (`build-dependencies`), not from the main binary. It's a giant static with a lot of strings, I'm not sure if it's guaranteed to *not* be included in the final binary if all you use is nvic_prio_bits.

Reply:
I seems it does not affect size. I removed metadata and setting priorities and checked with `cargo objcopy --bin app --release -- -O binary app.bin` before and after and it was the same size. Actually it was exactly the same size, I assumed it would be a little smaller because priority was not set. But did it twice and it indeed rebuilt.